ContentAwareGenerator.php

```diff
@@ -155,9 +155,9 @@ protected function getBestLocaleRoute(SymfonyRoute $route, $parameters)
      * @param array $parameters which should contain a content field containing
      *                          a RouteReferrersReadInterface object
      *
-     * @return SymfonyRoute the route instance
-     *
      * @throws RouteNotFoundException if no route can be determined
+     *
+     * @return SymfonyRoute the route instance
```

Comment:
not sure I like this ordening and afaik, Symfony core places throws as last as well.